sample/src/main/java/io/fotoapparat/sample/ActivityJava.java

```diff
@@ -97,30 +106,23 @@ protected void onCreate(Bundle savedInstanceState) {
         fotoapparat = createFotoapparat();
 
         takePictureOnClick();
-        focusOnLongClick();
         switchCameraOnClick();
         toggleTorchOnSwitch();
         zoomSeekBar();
     }
 
     private Fotoapparat createFotoapparat() {
-        return Fotoapparat
-                .with(this)
-                .into(cameraView)
-                .previewScaleType(ScaleType.CenterCrop)
-                .lensPosition(back())
-                .frameProcessor(new SampleFrameProcessor())
-                .logger(loggers(
-                        logcat(),
-                        fileLogger(this)
-                ))
-                .cameraErrorCallback(new CameraErrorListener() {
-                    @Override
-                    public void onError(@NotNull CameraException e) {
-                        Toast.makeText(ActivityJava.this, e.toString(), Toast.LENGTH_LONG).show();
-                    }
-                })
-                .build();
+        return new Fotoapparat(this, cameraView, focusView, back(), ScaleType
+                .CenterCrop, cameraConfiguration, new Function1<CameraException, Unit>() {
```

Comment:
we also kept `CameraErrorListener` for java users so they don't have to deal with the weird `return Unit.INSTANCE;`.